extensions/amp-selector/1.0/README.md

```diff
@@ -0,0 +1,287 @@
+
+#### Example: Import via npm
+
+[example preview="top-frame" playground="false"]
+
+Install via npm:
+
+```sh
+npm install @ampproject/bento-selector
+```
+
+```javascript
+import React from 'react';
+import { BentoSelector, BentoSelectorOption} from '@ampproject/bento-selector/react';
+import '@ampproject/bento-selector/styles.css';
+
+function App() {
+  return (
+    <BentoSelector aria-label="Option menu">
+      <BentoSelectorOption option="1">Option 1</BentoSelectorOption>
+      <BentoSelectorOption option="2">Option 2</BentoSelectorOption>
+      <BentoSelectorOption option="3">Option 3</BentoSelectorOption>
+      <BentoSelectorOption option="4">Option 4</BentoSelectorOption>
+    </BentoSelector>
+  );
+}
+```
+
+[/example]
+
+#### Layout and Style
+
+Each Bento component has a small CSS library you must include to guarantee proper loading without [content shifts](https://web.dev/cls/). Because of order-based specificity, you must manually ensure that stylesheets are included before any custom styles.
+
+```html
+<link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/amp-facebook-1.0.css">
+```
+
+Alternatively, you may also make the light-weight pre-upgrade styles available inline:
+
+```html
+<style data-bento-boilerplate>
+ BentoSelector{
+    display: block;
+    overflow: hidden;
+    position: relative;
+  }
+</style>
+```
+
+#### Attributes on `<BentoSelector>`
+
+**disabled, form, multiple, name**
+
+The attributes above behave the same way as they do on a standard HTML [`<select>`](https://developer.mozilla.org/en/docs/Web/HTML/Element/select) element.
+
+**keyboard-select-mode**
+
+The `keyboard-select-mode` attribute dictates the keyboard navigation behavior for options inside `<BentoSelector>`.
+
+<ul>
+  <li>
+    `none` (default): The tab key changes focus between items in the `<BentoSelector>`. The user must press enter or space to change the selection. Arrow keys are disabled.
+  </li>
+  <li>
+    `focus`: Tab key gives focus to `<BentoSelector>`. The user navigates between items with the arrow keys. Must press space or enter to change the selection.
+  </li>
+  <li>
+    `select`: Tab key gives focus to `<BentoSelector>`. The selection changes as the user navigates options with arrow keys.
+  </li>
+</ul>
+
+#### Attributes on `<BentoSelectorOption>`
+
+**option**
+
+Indicates that the option is selectable. If a value is specified, the contents of the value is submitted with the form.
+
+**disabled, selected**
+
+The attributes above behave the same way as they do on a standard HTML [`<option>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option) element.
+
+#### Actions
+
+The `BentoSelector` API allows you to perform the following actions:
+
+**selectBy(delta: number)**
+Closes the selector.
+
+```js
+api.selectBy(1); // Select next option in DOM sequence.
```

Comment:
should we specify that this `api` object comes from the `ref.current`? Or that's where I assume it comes from looking at storybook.